src/sections/HomePageBanner/HomePageBanner.module.css

```diff
@@ -1,156 +1,52 @@
 }
 
-.title_a {
-  color: var(--secondary-600-color);
-  font-feature-settings: "calt";
+.heading {
+  background: var(--green-gradient-100);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  width: 100%;
+  max-width: 42rem;
 }
 
-.arrow {
-  color: var(--tertiary-700-color);
-  margin-right: 1rem;
+.heading > span{
+   background: var(--white);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
 }
 
-.banner_left {
-   width: 100%;
-   flex: 1;
-   margin-bottom: 2rem;
-   justify-content: center;
-}
-
-
-.title {
-  color: var(--neural-950-color);
-  margin-bottom: 2rem;
+.description {
+  color: var(--primary-300);
+  margin: 1.5rem 0 3rem;
+  width: 100%;
+  max-width: 32.625rem;
 }
 
-
-.imageContainer {
-  flex: 1;
-  overflow: hidden;
-  position: relative;
-  display: none;
-  height: 100%;
-  width: 50vw;
-  align-items: center;
-  justify-content: start;
+.contents a,
+.contents a:hover {
+  color: var(--dark-green-700);
 }
-
-
- .imageContainer:before {
-   position: absolute;
-   content: "";
-   left: 9.625rem;
-   height: 568px;
-   top: 4.375rem;
-   width: 412px;
-   background-color: var(--secondary-300-color);
-   border-radius: 347.1875rem;
- }
-
-.image {
-  position: relative;
-  width: 90%;
-  max-width: 520px;
-  height: 408px;
-  margin-left: 6.1875rem;
-  margin-right: auto;
-}
-
-
-.list {
-  padding: unset;
-  margin-bottom: 2rem;
-}
-
-.list_item {
-  list-style: none;
-  margin-bottom: 1.25rem;
+.contents a {
+  background: var(--white);
+  border-radius: 2.5rem;
+  width: 100%;
+  max-width: 201px;
+  height: 64px;
   display: flex;
-  align-items: center;
-}
-
-.list_icon {
-  margin-right: 1.0625rem;
-}
-
-
-@media (min-width: 1024px) {
-  .container {
-    padding: unset;
-    height: 44.25rem;
-  }
-
-  .banner_left {
-    margin-bottom: 0;
-  }
-
-.banner_left > div {
-  margin-right: auto;
-  margin-left: auto;
-}
- 
-  .imageContainer {
-    display: flex;
-  }
-
-  .imageContainer:before {
-    left: 3.4rem;
-  }
-
-  .image {
-    margin-left: auto;
-  }
-  .title {
-    max-width: 28.6875rem;
-  }
-
-  .list {
-    width: 24.375rem;
-  }
-}
-
-@media (min-width: 1200px) {
-  .banner_left > div {
-    margin-right: 1rem;
-  }
-
-  .title {
-    max-width: 32.6875rem;
-  }
-}
-
-@media (min-width: 1280px) {
-  .imageContainer:before {
-    left: 7.5rem;
-  }
- 
-}
-
-@media (min-width: 1380px) {
-  .imageContainer:before {
-    left: 9.5rem;
-  }
-
-}
-
-@media (min-width: 1476px) {
-  .imageContainer:before {
-    left: 7rem;
-  }
-
-  .image {
-    margin-left: 4.1875rem;
-  }
+  justify-content: center;
+  box-shadow: 0px 18px 22px rgba(255, 255, 255, 0.08), 0px 7.51997px 9.19107px rgba(255, 255, 255, 0.0575083), 0px 4.02054px 4.91399px rgba(255, 255, 255, 0.0476886), 0px 2.25388px 2.75474px rgba(255, 255, 255, 0.04), 0px 1.19702px 1.46302px rgba(255, 255, 255, 0.0323114), 0px 0.498106px 0.608796px rgba(255, 255, 255, 0.0224916);
```

Comment:
can you approximate the PX of this box shadow please to make this more readable.. (so if it is 7.51997px change to 7px)

Also not sure if we can do the same with the colors transparency. To be honest, this is not really adding a strong shadow, so change values slightly is not going to affect this much!